### ISSUE TITLE
Resize nastiness in AB(X)/discrete interfaces

### ISSUE THREAD

**Issue by BrechtDeMan** [BrechtDeMan/WebAudioEvaluationTool#126]
When resizing the window after loading the interface, the volume slider disintegrates (the 'rounded rectangle frame' around it moves to somewhere else) and the 'submit' button disappears (AB/ABX). See screenshots. 

APE and MUSHRA are fine in this regard. 

![ABX](https://cloud.githubusercontent.com/assets/4103243/17292317/8c9c20ce-57e9-11e6-8117-acf5c35ca3b7.png)

![AB](https://cloud.githubusercontent.com/assets/4103243/17292320/947b3d0c-57e9-11e6-84bb-23e3ea702b8b.png)

![discrete](https://cloud.githubusercontent.com/assets/4103243/17292313/83dc2ca4-57e9-11e6-906a-24833ea9590f.png)

**Comment by nickjillings** [comment]
Yeah these are all caused because of the sheer number of movements that I try to do. I'll be stripping back a LOT of the CSS markup changes and using my new best friend, flex-boxes (how I managed to do some things before this I don't know).

**Comment by nickjillings** [comment]
Fixed for AB

**Comment by nickjillings** [comment]
These are now fixed in 104ccf0 (Dev_main)

**Comment by BrechtDeMan** [comment]
Still a bit of resize weirdness, particularly the 'Submit' button jumping aside. Discrete is fine.

Volume slider jumping down is probably okay, just a bit odd that the box doesn't follow. 

![ABX](https://cloud.githubusercontent.com/assets/4103243/25279359/796614ba-26a6-11e7-9b87-41d6526a9d7e.png)
![AB](https://cloud.githubusercontent.com/assets/4103243/25279360/798433fa-26a6-11e7-8d6b-8da555515c10.png)
![discrete](https://cloud.githubusercontent.com/assets/4103243/25279361/7988bd76-26a6-11e7-83ea-f8346c2694ed.png)

**Comment by nickjillings** [comment]
Yeah the d

**Comment by nickjillings** [comment]
Yeah the discrete has some stranger issues when it gets too small. More work there. The box should follow the AB(X), I found the typo that caused that issue so it's probably compounded error.
